projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.spec.ts

```diff
@@ -1,21 +1,124 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { QueryDetailsComponent } from './query-details.component';
+import { QueryListItem } from '../../models';
+import { By } from '@angular/platform-browser';
+import { MockRpxTranslatePipe } from 'projects/ccd-case-ui-toolkit/src/lib/shared/test/mock-rpx-translate.pipe';
```

Comment:
Imports incorrectly ordered and reference to entire folder not needed, should be ../lib at most

Reply:
@connorpgpmcelroy - I have now updated the code.

Reply:
still references entire ccd-case-ui-toolkit folder, should be relative link '../../'